tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -113,6 +114,43 @@ func newSession(credentialPath, credentialAccountID, region string) (*awssession
 	})
 }
 
+// retrievePrivateKey retrieves private key in format required for generating
+// public key using the privateKeyPath
+func (a *AwsProvider) retrievePrivateKey() (*rsa.PrivateKey, error) {
+	privateKeyBytes, err := ioutil.ReadFile(a.privateKeyPath)
+	if err != nil {
+		return nil, fmt.Errorf("failed to find private key from path: %v", a.privateKeyPath)
```

Comment:
Please include the actual error, there can be other reasons instead of the file not being missing that readfile will fail